contentcuration/contentcuration/models.py

```diff
@@ -366,7 +366,7 @@ def save(self, *args, **kwargs):
         # Detect if node has been moved to another tree
         if self.pk is not None and ContentNode.objects.filter(pk=self.pk).exists():
             original = ContentNode.objects.get(pk=self.pk)
-            if original.parent and original.parent_id != self.parent_id:
+            if original.parent and original.parent_id != self.parent_id and not original.parent.changed:
```

Comment:
Stop propagating through ancestors if the original parent has already been marked as changed